neuralmonkey/decoders/autoregressive.py

```diff
@@ -162,9 +162,13 @@ def decoding_w(self) -> tf.Variable:
 
     @tensor
     def decoding_b(self) -> tf.Variable:
+        if self.tie_embeddings:
+            return None
```

Comment:
Nemůže nastat případ, kdy chceme tie a zároveň biasy? Asi to neni při importu z tensor2tensor, ale spíš bych to viděl na nějakej switch v konstruktoru..

Reply:
Tedka z hlavy ti to nereknu, ale tipnul bych, ze v tom clanku o svazanych embeddingach by mohlo byt oduvodneni, proc to v tomhle pripade nechces s biasy.